packages/web/pages/index.tsx

```diff
@@ -107,3 +107,8 @@ const Home: NextPage = observer(function () {
 });
 
 export default Home;
+
+export const getServerSideProps: GetServerSideProps<HomeProps> = async () => {
```

Comment:
getServerSideProps or getStaticProps? I think we expect infrequent changes for the ad banners. If we decide to use getStaticProps, make sure to include a cache refresh